src/nspawn/nspawn-network.c

```diff
@@ -395,37 +395,49 @@ int remove_bridge(const char *bridge_name) {
 }
 
 static int parse_interface(const char *name) {
-        _cleanup_(sd_device_unrefp) sd_device *d = NULL;
         int ifi, r;
 
         r = parse_ifindex_or_ifname(name, &ifi);
         if (r < 0)
                 return log_error_errno(r, "Failed to resolve interface %s: %m", name);
 
-        if (path_is_read_only_fs("/sys") <= 0) {
-                char ifi_str[2 + DECIMAL_STR_MAX(int)];
+        return ifi;
+}
 
-                /* udev should be around. */
+int check_interface(const char *name) {
```

Comment:
check is such a generic name. If you split this, maybe call this network_interface_test_initialized(), because that's what it actually does? Also, given you export this function now, really say "network interface" rather than just "interface", to be explicit, given that "interface" is used in many contexts on computers...

Reply:
Went with `test_network_interface_initialized`.